catalogData/redis32/ha/3x/k8s/sentinel-proxy-service.json

```diff
@@ -0,0 +1,23 @@
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "app": "redis-sentinel-proxy"
+    },
+    "name": "redis-sentinel-proxy"
+  },
+  "spec": {
+    "ports": [
+      {
+        "name": "redis-sentinel-proxy",
+        "port": 6379,
+        "protocol": "TCP"
+      }
+    ],
+    "selector": {
```

Comment:
All the selectors here will need to be more specific. This selector will match all proxy pods for all service instances at the moment. We should update app labels and selectors so that each service only talks to the pods from the same instance. Check out https://github.com/18F/kubernetes-broker/blob/master/catalogData/redis28/standard/k8s/deployment.json#L18-L22 for an example.